Source/WebKit/Platform/IPC/ObjectIdentifierReferenceTracker.h

```diff
@@ -60,22 +61,8 @@ class ObjectIdentifierReference {
     {
         return m_identifier.isHashTableDeletedValue();
     }
-    template<typename Encoder> void encode(Encoder& encoder) const
-    {
-        encoder << m_identifier;
-        encoder << m_version;
-    }
-    template<typename Decoder> static std::optional<ObjectIdentifierReference> decode(Decoder& decoder)
-    {
-        std::optional<T> identifier;
-        std::optional<uint64_t> version;
-        decoder >> identifier
-            >> version;
-        if (!decoder.isValid())
-            return std::nullopt;
-        return ObjectIdentifierReference { WTFMove(*identifier), *version };
-    }
 private:
+    friend struct IPC::ArgumentCoder<ObjectIdentifierReference, void>;
```

Comment:
This is probably not necessary because there are accessors for the identifier and version.